### ISSUE TITLE
SA1009 (ClosingParenthesisMustBeSpacedCorrectly) reported inside interpolated string

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1006]
The following declaration results in an SA1009 warning:

```csharp
Type type = typeof(string);
string message = $"The type is: {type.ToString()}";
```

I believe an exception should be made to SA1009 such that no space is required between `)` and `}` in this code. Furthermore, I believe SA1009 should be reported if a space exists in this location.

**Comment by sharwell** [comment]
@Noryoko This could be a good one for you, since it directly conflicts with your recent work in #1025.

**Comment by Noryoko** [comment]
I'll take it.
